### PR TITLE
LPS-166923 baseline

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://issues.liferay.com/browse/LPS-166923

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java [hunk 1]
```diff
@@ -557,6 +561,90 @@ private void _testAddObjectRelationshipOneToMany(
...
+
+		objectRelationship =
+			_objectRelationshipLocalService.fetchObjectRelationship(
+				objectRelationship.getObjectRelationshipId());
+
+		reverseObjectRelationship =
+			_objectRelationshipLocalService.fetchReverseObjectRelationship(
+				objectRelationship, true);
+
+		Assert.assertNotNull(objectRelationship.getDBTableName());
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java:613]: @marcelabc I would move the `Assert.assertNotNull(objectRelationship.getDBTableName());` to be right after the `objectRelationship = ... `

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java [hunk 2]
```diff
@@ -166,6 +169,65 @@ public void addObjectRelationshipMappingTableValues(
...
+	@Override
+	public ObjectRelationship createManyToManyObjectRelationshipTable(
+			long userId, ObjectRelationship objectRelationship)
+		throws PortalException {
+
+		ObjectDefinition objectDefinition1 =
+			_objectDefinitionPersistence.findByPrimaryKey(
+				objectRelationship.getObjectDefinitionId1());
+
+		ObjectDefinition objectDefinition2 =
```
Comment (carolmariaabb) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java:182]: @marcelabc I would remove the empty line here

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java [hunk 3]
```diff
@@ -557,6 +561,90 @@ private void _testAddObjectRelationshipOneToMany(
...
+		Assert.assertEquals(
+			StringPool.BLANK, objectRelationship.getDBTableName());
+
+		ObjectRelationship reverseObjectRelationship =
+			_objectRelationshipLocalService.fetchReverseObjectRelationship(
+				objectRelationship, true);
+
+		Assert.assertNotNull(reverseObjectRelationship);
+
+		Assert.assertEquals(
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java:597]: @marcelabc I would remove the empty line here

**Comment by marcelabc** [comment]
Hey @carolmariaabb,
I made all the changes, but I would like to ask you to review again please, mainly to the point that the method updateObjectRelationship was being called twice.

thank you!

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Review by carolmariaabb — APPROVED** [review]

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:reevaluate:1257401_5332

**Comment by marcelabc** [comment]
send to bchan: https://github.com/brianchandotcom/liferay-portal/pull/126077

**Pull request closed without merging**